hnn_core/tests/test_dipole.py

```diff
@@ -48,6 +48,23 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # average an n_of_1 dipole list
+    single_dpl_avg = average_dipoles([dipole])
+    for dpl_key in single_dpl_avg.data.keys():
+        np.allclose(dipole_read.data[dpl_key],
+                    single_dpl_avg.data[dpl_key], rtol=0, atol=0.000051)
+
+    # average dipole list with one dipole object and a zero dipole object
+    dpl_null = dipole_read
+    dpl_null.data['agg'] = np.zeros(len(dpl_null.data['agg']))
+    dpl_null.data['L2'] = np.zeros(len(dpl_null.data['L2']))
+    dpl_null.data['L5'] = np.zeros(len(dpl_null.data['L5']))
```

Comment:
why not just use the constructor: https://jonescompneurolab.github.io/hnn-core/dev/generated/hnn_core.dipole.Dipole.html#hnn_core.dipole.Dipole

```py
n_times = len(dipole_read)
dpl_zeros = Dipole(np.zeros(n_times, ), np.zeros(n_times, 3))
```

Reply:
@jasmainak I like that. Much cleaner. Will go make the changes.